Bhagyashri/first_assignment_17-july-2019/css/style.css

```diff
@@ -0,0 +1,114 @@
+
+body{
+background-color:#71D0C5;
+padding:0 100px;
+margin:0;
+}
+.first-sect1{
+	   border-top:0;
```

Comment:
we don't need border to set to zero as it is already zero.

Reply:
okay